crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -69,8 +74,47 @@ fn nonce_storage_key<T: AddressBytes>(address: T) -> String {
 pub(crate) trait StateReadExt: StateRead + crate::assets::StateReadExt {
     #[instrument(skip_all)]
     async fn get_account_balances(&self, address: Address) -> Result<Vec<AssetBalance>> {
+        let ibc_balances = self
```

Comment:
```suggestion
        let ibc_prefixed_balances = self
```

since there's the native asset, which is not ibc'd, but can be an ibc-prefixed denom